docs/timelines/creating-a-timeline.md

```diff
@@ -0,0 +1,99 @@
+---
+id: creating-a-timeline
+title: Creating a Timeline
+sidebar_label: Creating a Timeline
+---
+
+You can create a timeline by using Live Record mode, by entering triggers manually at specified times, or a combination of the two.
+
+
+
+Programming a Timeline using Live Record
+----------------------
+
+Live record is useful to capture a busked show, where you are operating the playbacks manually in time with music or other live cues. You can do a live record multiple times on the same timeline to build up a more complex show.
+
+>  Recall of Palettes will not be recorded using timeline Live Record. You need to save looks to a playback to use them on a timeline.
+
+This is how you program a timeline using Live Record, from playbacks which you have previously programmed:
+
+1. If you want to use internal timer rather than feeding in a timecode source, ensure the "Timecode Disconnect" \{Link\} button is selected to show a broken link.
+
+2. Press \<Record\>, \[Timeline\], then the **Select button** of the fader where you want to store the timeline. The Timeline View window will automatically open.
```

Comment:
We've added a few shorter ways to do this that are probably worth a mention.

- Double tap a track on the sidebar enters live record right away.
- Clicking play using internal timecode automatically starts live record, you don't have to press "start live record" first
-